.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dump.rdb
+dist
```

Comment:
Can you please tweak tsconfig so that it does not  produce any `dist` files?

Reply:
Nevermind. I see that we are producing declaration files in `dist/` in other repositories too, e.g. https://github.com/CheckerNetwork/spark-api/blob/main/tsconfig.json

Let's be consistent and do the same thing we do elsewhere.